distributed/cli/dask_worker.py

```diff
@@ -175,6 +175,30 @@
 @click.option(
     "--dashboard-prefix", type=str, default="", help="Prefix for the dashboard"
 )
+@click.option(
+    "--lifetime",
+    type=str,
+    default="",
+    help="If provided, shut down the worker after this duration. "
+    "Note that if combined with --nanny (default) "
+    "the worker will restart after this time",
```

Comment:
Even with lifetime_restart to false as default?

Reply:
Thanks.  I've removed this.  I think that it's now clear with the `--lifetime-restart` keyword just below.